doc/doxygen/src/debugging-aids.md

```diff
@@ -1,5 +1,33 @@
 # Debugging Tools {#debugging-tools}
```

Comment:
Not sure if this is the best subsection for the coverage documentation.

Maybe it makes sense to rename this to `Developer Tooling` or something along those lines?